src/libexpr/parser.y

```diff
@@ -755,11 +758,11 @@ SourcePath EvalState::findFile(SearchPath & searchPath, const std::string_view p
 {
     for (auto & i : searchPath) {
```

Comment:
Can we just use [structured binding](https://en.cppreference.com/w/cpp/language/structured_binding) here, instead of introducing a new type?

i.e. `for(auto &[prefix, path] : searchPath)`